server/server.go

```diff
@@ -544,7 +541,6 @@ func (server *Server) loadConfig(configurations configs, globalConfiguration Glo
 	redirectHandlers := make(map[string]negroni.Handler)
 	backends := map[string]http.Handler{}
 	backendsHealthcheck := map[string]*healthcheck.BackendHealthCheck{}
-	backend2FrontendMap := map[string]string{}
```

Comment:
Because the backend to frontend map was only used by the old logger.  The new framework, added during a previous PR, uses SaveFrontend and SaveBackend to achieve the same thing.